Source/Core/Utils.jsl

```diff
@@ -694,6 +694,25 @@ ut delete symbols difference = Function({before, after, closeWindows=0},
 	::ut nullptr;
 );
 
+// Variable: ut lang
+// Code for the current language (en, de, es, fr, it, ja, ko, zh-CN)
+ut lang = System Service(Get Language Code);
```

Comment:
This should probably be an `Expr` in case the language is changed after `Hamcrest` has started.